packages/aragon-wrapper/src/index.js

```diff
@@ -898,14 +898,14 @@ export default class Aragon {
           },
           { ethNode: this.web3.currentProvider }
         )
-
-        try {
-          const processed = await this.postprocessRadspecDescription(description)
-          description = processed.description
-          annotatedDescription = processed.annotatedDescription
-        } catch (err) { console.log(err) }
```

Comment:
Wasn't sure if there was a particular exception being caught here; as far as I can tell, `postprocessRadspecDescription()` shouldn't throw if you provide it a string

Reply:
You are right, forgot to remove it after testing